Research/Research/RSDColorRules.swift

```diff
@@ -178,21 +178,37 @@ open class RSDColorRules  {
 
     /// Color for text throughout the app.
     ///
-    /// - Default:
+    /// - Default: (version 1)
     ///     If the background uses light style
-    ///         then `white`
-    ///         else `veryDarkGray`
+    ///         then `text.light`
+    ///         else if detail uses `text.medium` else `text.dark`
```

Comment:
Is there something missing in this sentence or am I missing something?

Reply:
I was trying to figure out a way of capturing the rules using pseudocode. It's no longer "veryDarkGray" and now there is a color family with 3 colors (most have 5 but this one has 3). Not really sure how to describe the rules without using code.

Reply:
Understood, but I'm having a bit of trouble parsing the pseudocode. How is the decision to use `text.medium` vs `text.dark` made? Or should the last else in this line be a then? Or...?